packages/pwa-kit-react-sdk/src/ssr/universal/errors.js

```diff
@@ -11,7 +11,7 @@ export class HTTPError extends Error {
         this.constructor = HTTPError
         this.__proto__ = HTTPError.prototype
         this.message = message
-        this.status = status
+        this.status = Number(status)
```

Comment:
We need to parse this string to split at the first space character and properly coerce the left-hand side of the split to a number

Reply:
I removed this casting here and updated the place in Auth where a string status can be passed in (which was just the getToken call)